pkg/prompts/prompts.go

```diff
@@ -286,7 +362,7 @@ func (*realPrompter) CapturePChainAddress(promptStr string, network models.Netwo
 	return prompt.Run()
 }
 
-func (*realPrompter) CaptureAddress(promptStr string) (common.Address, error) {
+func (*realPrompter) CaptureAddress(promptStr string, arg any) (any, error) {
```

Comment:
the arg is not needed, should not be included

Reply:
I think  it is, as already commented above, because the signatures are different. Like this, it becomes an optional param.